exercise/tasks.py

```diff
@@ -0,0 +1,54 @@
+import logging
+from time import sleep
+
+from celery import current_task
+from django.db.models import Q
+from aplus.celery import app
+from .exercise_models import BaseExercise
+
+logger = logging.getLogger('aplus.exercise')
+
+@app.task
+def regrade_exercises(exerciseid: int, uri_prefix: str, regrade_type: str) -> None:
+    exercise = BaseExercise.objects.get(pk=exerciseid)
+
+    qs = exercise.submissions\
+        .defer("feedback", "submission_data", "grading_data")
+
+    if regrade_type == 'incomplete':
+        qs = qs.filter(Q(status='INITIALIZED') | Q(status='WAITING') | Q(status='ERROR'))
+
+    total = qs.count()
+
+    # FIXME: "Simulated" type is used for testing with arbitrarily higher loads.
+    # It will be removed from the final PR, along with the "iterations" variable.
+    iterations = 1
+    if regrade_type == 'simulated':
+        iterations = 15
+        total = total * iterations
+
+    count = 0
+    while iterations > 0:
+        for i in qs:
```

Comment:
`i` is a submission so shouldn't the variable be named `submission`? `i` is often associated with an integer variable in loops.

-> use `submission`